recipes/coverm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.0.alpha7" %}
+{% set name = "coverm" %}
+{% set sha256 = "0e20493db934956a9d32cabf9ae28dabb7cc5ed2fb2b7a19dab74cdd4d99e17a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  url: https://github.com/wwood/{{ name }}/archive/v0.2.0-alpha7.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - rust >=1.19
+    - {{ compiler('c') }}
+    - clangdev
+    - {{ compiler('c') }}  # [not osx]
+    - {{ compiler('cxx') }}  # [not osx]
+    - clang_osx-64 # [osx]
+    - clangxx_osx-64 # [osx]
+    - pkg-config  # [osx]
```

Comment:
What's the reason for using the selectors? clang_osx-64 is what will get used if you use `{{ compiler('c') }}` on OSX anyway.

Reply:
Thanks for the review. Some kind of clang-related input is required because otherwise compilation of the rust-htslib dependency fails - see this earlier attempt: https://circleci.com/gh/bioconda/bioconda-recipes/65914

I have to admit my fix to that problem might have been a little ham-fisted - I just copied from the rust-bio-tools meta.yaml which presumably had similar problems. Knowing that, would you still suggest removing inputs?

Thanks.